app/models/server/models/Rooms.js

```diff
@@ -535,6 +535,10 @@ export class Rooms extends Base {
 		return this._db.find(query, options);
 	}
 
+	findTeamMainRooms(options) {
+		return this._db.find({ teamMain: { $exists: true, $eq: true } }, options);
```

Comment:
just a small change as both are equivalent but this is much easier to read:
```suggestion
		return this._db.find({ teamMain: true }, options);
```